storage/mounts_test.go

```diff
@@ -333,21 +333,23 @@ func TestMountCRD(t *testing.T) {
 			Response:     apierr.NotFound("nope"),
 		},
 	}, func(ctx context.Context, client *common.DatabricksClient) {
-		r := ResourceMount()
+		r := ResourceMount().ToResource()
 		d := r.TestResourceData()
+		d.Set("name", "a name")
```

Comment:
The actual resource definition adds a "preprocessing" step that checks whether the `name` and `uri` attributes are set.